change/@fluentui-react-utilities-7013f126-ca0d-4752-b4a9-e9d912be491a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31636&commitId=3657f90e8ee067ef48f566e1328271541dcf237a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>